Makefile

```diff
@@ -54,7 +54,11 @@ tests-toy: filecheck-toy pytest-toy
 tests-marimo:
 	@for file in docs/marimo/*.py; do \
 		echo "Running $$file"; \
-		python3 "$$file" || exit 1; \
+		error_message=$$(python3 "$$file" 2>&1) || { \
+			echo "Error running $$file"; \
+			echo "$$error_message"; \
+			exit 1; \
```

Comment:
Can we make it s.t. it does not fail on the first failing one, but runs all of them and then reports the errors or something?

Reply:
https://github.com/xdslproject/xdsl/issues/2841